packages/create-platformatic-db/src/create-gitignore.mjs

```diff
@@ -0,0 +1,31 @@
+import { isFileAccessible } from './utils.mjs'
+import { writeFileSync } from 'node:fs'
+import { join } from 'node:path'
+
+const gitignore = `
+# dotenv environment variable files
+.env
+
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+.pnpm-debug.log*
+
+# Dependency directories
+node_modules/
```

Comment:
Can you add `dist` because it's a default folder we use to build ts. I'm not sure about the `build` folder.
`.DS_Store` for mac os users who don't know how to put .gitignore into the root folder.